README.md

```diff
@@ -1,92 +1,199 @@
+
+Samples is a non official part of a project, that contains examples of how DAAL can be used in different applications.
+
+**DAAL Spark Samples is a good example.**
+DAAL provides scala / java interfaces that match Apache Spark MlLib API and use DAAL solvers under the hood. This implementation allows you to get a 3-18X increase in performance compared to default Apache Spark MLlib.
+
+  ## Examples
+
+Except C++ and Python API DAAL also provide API for C++ SYCL and Java languages. Check out tabs below for more examples.
+- [C++](https://github.com/intel/daal/tree/master/examples/daal/cpp)
+- [C++ SYCL*](https://github.com/intel/daal/tree/master/examples/daal/cpp_sycl)
+- [Java](https://github.com/intel/daal/tree/master/examples/daal/java)
 
 Data Examples for different computation modes:
 
-- [Batch](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/batch)
-- [Distributed](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/distributed)
-- [Online](https://github.com/oneapi-src/oneDAL/tree/master/examples/data/online)
+- [Batch](https://github.com/intel/daal/tree/master/examples/daal/data/batch)
+- [Distributed](https://github.com/intel/daal/tree/master/examples/daal/data/distributed)
+- [Online](https://github.com/intel/daal/tree/master/examples/daal/data/online)
 
 ## Documentation
-
-- [Get Started](http://oneapi-src.github.io/oneDAL/getstarted.html)
-- [System Requirements](https://software.intel.com/en-us/articles/system-requirements-for-oneapi-data-analytics-library#)
-- [oneDAL documentation](http://oneapi-src.github.io/oneDAL/)
-- [Specifications](https://spec.oneapi.com/versions/latest/elements/oneDAL/source/index.html)
+- [Get Started](http://intel.github.io/daal/getstarted.html)
+- [oneDAL documentation](http://intel.github.io/daal/)
+- [Specifications](https://spec.oneapi.com/oneDAL/index.html)
 - [Release Notes](https://software.intel.com/en-us/articles/oneapi-dal-release-notes)
-- [Known Issues](https://oneapi-src.github.io/oneDAL/notes/known_issues.html)
 
-## API
+## Installation
+
+You can install oneDAL:
 
-Intel(R) DAAL [provides](https://software.intel.com/en-us/articles/daal-api-reference) downloadable API References for C++, Python, and Java.
+- from [oneDAL home page](https://software.intel.com/en-us/oneapi/onedal) as a part of Intel&reg; oneAPI Base Toolkit.
+- from [GitHub\*](https://github.com/intel/daal/releases).
 
-You can also use [daal4py](https://github.com/IntelPython/daal4py), a simplified Python API to Intel(R) DAAL that allows fast usage of the framework suited for Data Scientists or Machine Learning users.
+## Installation from Source
+See [Installation from Sources](INSTALL.md) for details.
+
+## Technical Preview Features
+
+Technical preview features are introduced to gain early feedback from developers. A preview feature is subject to change in the future releases. Using a preview feature in a production code base is therefore strongly discouraged.
```

Comment:
```suggestion
Technical preview features are introduced to gain early feedback from developers. A technical preview feature is subject to change in the future releases. Using a technical preview feature in a production code base is therefore strongly discouraged.
```

After that line, please also add the following text proposed by @averbukh: 

```

In C++ APIs, technical preview features are located in `daal::preview` and `onedal::preview` namespaces. In Java APIs, technical preview features are located in packages that have the `com.intel.daal.preview` name prefix.

```

Reply:
Added